xpti/include/xpti/xpti_data_types.h

```diff
@@ -376,26 +376,32 @@ enum class trace_point_type_t : uint16_t {
   /// Used to trace function call begin and its arguments.
   function_with_args_begin = XPTI_TRACE_POINT_BEGIN(14),
   /// Used to trace function call end.
-  function_with_args_end = XPTI_TRACE_POINT_END(15),
+  function_with_args_end = XPTI_TRACE_POINT_END(14),
```

Comment:
Why isn't it an ABI break?

Reply:
Good catch! As it was implemented, it is incorrect as scoped notify will fail. This change was attempting to fix that issue. Looks like we need to revert back until ABI breakage is allowed and handled it separately until then.